### ISSUE TITLE
What would the equivalent of NORM.DIST be?

### ISSUE THREAD

**Issue by brandonros** [simple-statistics/simple-statistics#544]
https://support.microsoft.com/en-us/office/norm-dist-function-edb1cc14-a21c-4e53-839d-8082074c9f8d

```
NORM.DIST(x,mean,standard_dev,cumulative)

Returns the normal distribution for the specified mean and standard deviation. This function has a very wide range of applications in statistics, including hypothesis testing.

The NORM.DIST function syntax has the following arguments:

X     Required. The value for which you want the distribution.

Mean     Required. The arithmetic mean of the distribution.

Standard_dev     Required. The standard deviation of the distribution.

Cumulative     Required. A logical value that determines the form of the function. If cumulative is TRUE, NORM.DIST returns the cumulative distribution function; if FALSE, it returns the probability density function.
```

**Comment by dsaxton** [comment]
@brandonros If it's the cumulative version you're interested in you can use `ss.cumulativeStdNormalProbability`, but instead of plugging in `x` directly use `(x - mean) / standard_dev`. For the density function I'm not finding anything in `simple-statistics` (@tmcw please correct me if I'm wrong here) but you could always use the formula: https://en.wikipedia.org/wiki/Normal_distribution

**Comment by shrddr** [comment]
Density function missing is quite unexpected for me

**Comment by tmcw** [comment]
That's true, there's no equivalent of NORM.DIST in simple-statistics at the moment. I'd welcome such a PR, and it looks like there are some examples in JavaScript, like https://github.com/RichieAHB/normal-distribution

**Comment by MohammedAlkindi** [comment]
Current state on `main` (v7.10.0), with measured numbers, since this is still open and the answer above is now partly improvable.

**Cumulative — `NORM.DIST(x, mean, sd, TRUE)`** has two routes today:

```js
const z = (x - mean) / sd;                     // Excel's doc example: x=42, mean=40, sd=1.5
ss.cumulativeStdNormalProbability(z);          // 0.9082
0.5 * (1 + ss.errorFunction(z / Math.SQRT2));  // 0.9087887756126962
```

Excel documents `0.9087888` for that example. `cumulativeStdNormalProbability` is a lookup in the 4-decimal `standardNormalTable` with z rounded to the nearest 0.01, so its worst error over z ∈ [−4, 4] is **2.0e-3** (measured against numerically integrated ground truth); the `errorFunction` route is within **4.2e-8** on the same sweep. For Excel-matching output, use the `errorFunction` form.

**Density — `NORM.DIST(x, mean, sd, FALSE)`** still has no library function, as established above. The formula is one line:

```js
Math.exp(-((x - mean) ** 2) / (2 * sd * sd)) / (sd * Math.sqrt(2 * Math.PI)); // 0.10934004978399577
```

(Excel documents `0.10934`.) The inverse direction exists as `ss.probit`, standard normal only.

So the remaining gap for a true NORM.DIST equivalent is the density plus a mean/sd-parameterized wrapper — the PR invitation above still stands.
